alws/crud/release.py

```diff
@@ -135,7 +86,7 @@ async def get_release_plan(db: Session, build_ids: typing.List[int],
             'repositories': prod_repos
         }
 
-    oracle_response = await OracleClient(settings.packages_oracle_host).post(
+    oracle_response = await BeholderClient(settings.packages_oracle_host).post(
```

Comment:
Right now in `alws.config` we have vars for oracle and beholder, doesn't they equal?